lua/neogit/lib/git/init.lua

```diff
@@ -0,0 +1,58 @@
+local cli = require("neogit.lib.git.cli")
+local notif = require("neogit.lib.notification")
+local input = require("neogit.lib.input")
+
+local M = {}
+
+M.create = function(directory, sync)
+  sync = sync or false
+
+  if sync then
+    cli.init.args(directory).call_sync()
+  else
+    cli.init.args(directory).call()
+  end
+end
+
+M.init_repo = function()
+  local directory = vim.fn.input {
+    prompt = "Create repository in: ",
+    text = "",
+    cancelreturn = "",
+    completion = "dir",
+  }
+  if directory == "" then
+    return
+  end
+
+  -- git init doesn't understand ~
+  if string.find(directory, "~") == 1 then
```

Comment:
Perhaps we should use vim.fn.fnamemodify(directory, ":p") instead to resolve all of this

Reply:
yeah that's much better.. I've changed that in the latest commit.